### PR TITLE
Adding LANDSCAPE_ENVIRONMENT and LANDSCAPE_NAME

### PR DESCRIPTION
**Pull request by liorfr**

### CODE REVIEW

File: src/unleash_client_wrapper.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export async function initializeUnleashClient(extensionName: string, serverArgs:
     refreshInterval: serverArgs.ftServerInterval,
     url: serverArgs.ftServerEndPoint,
     strategies: customStrategies,
+    //customHeaders: {"authorization" : ""},
```
Comment (allacherkes) [inline src/unleash_client_wrapper.ts:27]: why is this comment here?

File: src/appstudio_context.ts [hunk 2]
```diff
@@ -39,18 +41,22 @@ function extractCfSubAccountAndWs(wsBaseUrlString: string, context: AppStudioMul
 export function createContextObject(): AppStudioMultiContext {
   // get the user name from the env
   const userName = getEnv(USER_NAME, "Feature toggle env USER_NAME was NOT found in the environment variables");
+  // get the environment from the env
+  const environment = getEnv(LANDSCAPE_ENVIRONMENT, "Feature toggle env LANDSCAPE_ENVIRONMENT was NOT found in the environment variables");
```
Comment (allacherkes) [inline src/appstudio_context.ts:45]: They exposed environment variables in small letters and you have it in big. Doesnt it fail the flow?

File: src/appstudio_context.ts [hunk 3]
```diff
@@ -39,18 +41,22 @@ function extractCfSubAccountAndWs(wsBaseUrlString: string, context: AppStudioMul
...
+  // get the landscape from the env
+  const landscape = getEnv(LANDSCAPE_NAME, "Feature toggle env LANDSCAPE_NAME was NOT found in the environment variables");
```
Comment (allacherkes) [inline src/appstudio_context.ts:47]: see above

File: src/appstudio_context.ts (hunk 2 above)
Comment (liorfr) [reply src/appstudio_context.ts:45]: I have talked to them and it was changed to upper case

File: src/appstudio_context.ts (hunk 3 above)
Comment (liorfr) [reply src/appstudio_context.ts:47]: same

File: src/unleash_client_wrapper.ts (hunk 1 above)
Comment (liorfr) [reply src/unleash_client_wrapper.ts:27]: it is needed for development, for local testing...

File: src/appstudio_context.ts (hunk 2 above)
Comment (allacherkes) [reply src/appstudio_context.ts:45]: yep, I saw :)

**Pull request merged**